client/transactions/list/test/index.tsx

```diff
@@ -3,20 +3,20 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import * as React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
 import user from '@testing-library/user-event';
 import apiFetch from '@wordpress/api-fetch';
 import { getQuery, updateQueryString } from '@woocommerce/navigation';
 import { useUserPreferences } from '@woocommerce/data';
-import { getUserTimeZone } from 'wcpay/utils/test-utils';
```

Comment:
This was not an external dependency - moved to the `Internal dependencies` block